static/src/js/util/retrievals.js

```diff
@@ -108,12 +108,31 @@ export const prepareRetrievalParams = (state) => {
 
     const params = buildGranuleSearchParams(preparedParams)
 
+    const allVariables = accessMethods[selectedAccessMethod].variables
```

Comment:
I'm surprised eslint let this go through, you should destructure variables off of the object